server/db/Cart.js

```diff
@@ -16,23 +17,32 @@ Cart.init(
   { sequelize: db, modelName: 'carts' }
 );
 
+// Finds the Cart and associated products, ordered by name
+Cart.getWithProducts = async (cartId) => {
+  try {
+    const cart = Cart.findByPk(cart.id, {
+      include: {
+        model: CartProduct,
+        include: Product,
+        separate: true,
+        order: [[Product, 'name', 'ASC']],
+      },
+    });
+    return cart;
+  } catch (err) {
+    console.error(err);
+    throw err;
+  }
+};
```

Comment:
Added this as a class method, so our server can quickly get the Cart + Products ordered by name by passing in a cart id